homeassistant/components/light/limitlessled.py

```diff
@@ -33,32 +37,54 @@
 from homeassistant.util.color import color_RGB_to_xy
 
 _LOGGER = logging.getLogger(__name__)
-REQUIREMENTS = ['ledcontroller==1.0.7']
+REQUIREMENTS = ['ledcontroller==1.1.0']
 
 
 def setup_platform(hass, config, add_devices_callback, discovery_info=None):
     """ Gets the LimitlessLED lights. """
     import ledcontroller
 
-    led = ledcontroller.LedController(config['host'])
+    # Handle old configuration format:
+    bridges = []
+    if 'bridges' in config:
+        bridges = config['bridges']
+    else:
+        bridges.append(config)
+
+    i = 0
+    for bridge in bridges:
+        bridge['id'] = i
+        i = i + 1
+
+    pool = ledcontroller.LedControllerPool([x['host'] for x in bridges])
 
     lights = []
-    for i in range(1, 5):
-        if 'group_%d_name' % (i) in config:
-            lights.append(LimitlessLED(led, i, config['group_%d_name' % (i)]))
+    for bridge in bridges:
+        for i in range(1, 5):
+            name_key = 'group_%d_name' % i
+            type_key = 'group_%d_type' % i
+            if name_key in bridge:
+                if type_key in bridge and bridge[type_key] == 'white':
+                    lights.append(LimitlessLEDWhite(pool, bridge['id'], i,
+                                                    bridge[name_key]))
+                else:
+                    lights.append(LimitlessLED(pool, bridge['id'], i,
+                                               bridge[name_key]))
 
     add_devices_callback(lights)
 
 
+# pylint: disable=too-many-instance-attributes
 class LimitlessLED(Light):
```

Comment:
It looks like the only difference between these two classes is in the `turn_on` method and the `brightness` and `color_xy` properties. Could you either rewrite this using inheritance or a boolean value to track if it is a white or colored light?